tests/legacy/test_resource_warnings.py

```diff
@@ -17,10 +19,21 @@ def test_task_double_close(self, init_kwargs):
             with nidaqmx.Task(**init_kwargs) as task:
                 task.close()
 
+    @pytest.mark.library_only
     def test_unclosed_task(self, init_kwargs):
```

Comment:
Hmm, these tests actually leak tasks. Perhaps they should save the interpreter and task handle and call clear_task at the end.
![image](https://github.com/ni/nidaqmx-python/assets/17914399/4dee59b4-a901-4693-ad1d-7ce169c72d3a)

Reply:
Closed all the leaking tasks.

Reply:
Oops, I should have updated/closed this when I realized we can still call `task.close()` after `task.__del__()`.